core/src/main/java/org/apache/druid/java/util/metrics/JvmMonitor.java

```diff
@@ -40,10 +42,12 @@
 
 public class JvmMonitor extends FeedDefiningMonitor
 {
+  private static final Logger log = new Logger(JvmMonitor.class);
+
   private final Map<String, String[]> dimensions;
   private final long pid;
 
-  private final GcCounters gcCounters = new GcCounters();
+  final GcCounters gcCounters;
```

Comment:
`@VisibleForTesting`?

Reply:
Please annotate this field as `@Nullable`.

Reply:
done